tests/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixerTest.php

```diff
@@ -2566,6 +2566,48 @@ class Cl {
                 }
                 EOD,
         ];
+
+        yield '@param for future parameter with option disabled' => [
+            <<<'EOD'
+                <?php
+                /**
+                 */
+                function foo(array $bundleConfig, \ReflectionClass $bundle, ContainerBuilder $container, /* string $bundleDir1 = null, */ $foo1/** , string $bundleDir2 = null */, $foo2 /* , mixed $bundleDir3 */) {}
+                EOD,
+            <<<'EOD'
+                <?php
+                /**
+                 * @param array   $bundleConfig
+                 * @param ?string $bundleDir1
+                 * @param ?string $bundleDir2
+                 * @param mixed   $bundleDir2
```

Comment:
I think you meant
```suggestion
                 * @param mixed   $bundleDir3
```
(Same for the other test)

Reply:
good eye, fixed